components/automate-ui/src/app/pages/+compliance/+profile/+profile-details/profile-details.component.spec.ts

```diff
@@ -30,30 +30,16 @@ describe('ProfileDetailsComponent', () => {
     username: 'TestUser'
   };
 
-  const initialState = {
-    router: {
-      state: {
-        url: '/',
-        params: {},
-        queryParams: {},
-        fragment: ''
-      }
-    },
-    clientRunsEntity: fromClientRuns.ClientRunsEntityInitialState,
-    notifications: fromNotifications.InitialState,
-    layout: fromLayout.InitialState
-  };
```

Comment:
This initial state is not actually providing any customization. That was a smell that indicated it was not even needed, so this commit removes all those.